solidity/src/base/Constants.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: UNLICENSED
+// This is licensed under the Cryptographic Open Software License 1.0
+pragma solidity ^0.8.28;
+
+uint32 constant FILTER_EXEC_NUM = 0;
+
+uint32 constant COLUMN_EXPR_NUM = 0;
+uint32 constant EQUALS_EXPR_NUM = 1;
+uint32 constant LITERAL_EXPR_NUM = 2;
+
+uint32 constant BIGINT_TYPE_NUM = 0;
+
+uint256 constant TOO_FEW_CHALLENGES = 0x700caebe_00000000_00000000_00000000_00000000_00000000_00000000_00000000;
```

Comment:
Question: Are these values hashes? How are they obtained?

Reply:
Yeah, they are hashes of the error signatures. So, this is the keccak hash of `TooFewChallenges()`.

Reply:
See my new solution in #512. That should resolve this in the smaller PRs.